### PR TITLE
Update Subscription Services PN

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

**Comment by flodolo** [comment]
@jejohnsonmoz @peiying2 

The initial content of the PR should match the status of #1996 before it was merged by mistake. I'm going to add other commits with some of the straight-forward fixes, but Jennifer won't be able to make changes here (this is a branch in this repo, and she doesn't have access).

She can comment, and @peiying2 will need to make those changes.

**Comment by flodolo** [comment]
Corrections here: https://github.com/mozilla/legal-docs/pull/2000/commits/baa8e6f7d7d72be0a38ccbc4595f71c6ecbb8814

I'll comment where questions were still open for @jejohnsonmoz

File: en/subscription_services_privacy_notice.md [hunk 1]
```diff
@@ -1,55 +1,68 @@
...
+
+* [HaveIbeenpwned.](https://haveibeenpwned.com/) HaveIbeenpwned receives a hashed version of your email address in order to perform a scan of where your personal data may have been compromised. HaveIbeenpwned keeps this personal data to continuously scan for new breaches; you can request removal of your email address from HaveIbeenpwned via their [opt-out mechanism](https://haveibeenpwned.com/OptOut).
+  
+* [OneRep.](https://onerep.com/) If you are located in the United States and have a Monitor Premium Subscription, OneRep receives your first and last name, email address, phone number, physical address and date of birth in order to scan data broker sites to find your personal data and request its removal. OneRep keeps your personal data until you end your Monitor subscription in order to check whether your information shows up on additional sites, or has reappeared on the sites you’ve already been removed from.
```
Comment (flodolo) [inline en/subscription_services_privacy_notice.md:42]: Open question: Isn't it going to be Monitor Plus by the time this document is valid? Also, I think subscription should be lowercase here.

File: en/subscription_services_privacy_notice.md [hunk 2]
```diff
@@ -1,55 +1,68 @@
...
 
 ## At Mozilla, we design products with your privacy in mind.
 
 __Mozilla VPN__ protects your device’s internet connections. Mozilla partners with Mullvad to privately and securely encrypt your internet traffic.
 
-__Firefox Relay__ protects your identity with secure email and phone masking. Our secure and easy to use email and phone masks help keep your identity private so you can sign up for new accounts anonymously, stop spam texts and junk calls, and get only the emails you want in your inbox.  
+__Mozilla Monitor__ can monitor to see if any of your accounts have been in known data breaches. In some regions, it can also help you scan for and remove your information from places around the web. 
 
-__Firefox Monitor__ can monitor to see if any of your accounts have been in known data breaches. See [Firefox Monitor Terms and Privacy](https://www.mozilla.org/privacy/firefox-monitor/).
+__Firefox Relay__ protects your identity with secure email and phone masking. Our secure and easy to use email and phone masks help keep your identity private so you can sign up for new accounts anonymously, stop spam texts and junk calls, and get only the emails you want in your inbox.
```
Comment (flodolo) [inline en/subscription_services_privacy_notice.md:12]: Open question: I think this needs to add "In some regions" here, similarly to Monitor. Phone masking is only available in US.

**Review by peiying2 — CHANGES_REQUESTED** [review]
@jejohnsonmoz could you go over the comments and see if you agree. Because I accidentally merged the PR with pending changes still needed, you can not make any changes to this PR, but you can add your comment on how you want to change it, so I or we can do it, so we can wrap this up before sending it out to be localized.

File: en/subscription_services_privacy_notice.md (hunk 2 above)
Comment (jejohnsonmoz) [reply en/subscription_services_privacy_notice.md:12]: Confirmed with Luke Crouch. Can we please update to "__Firefox Relay__ protects your identity with secure email and, in some regions, phone masking."?

File: en/subscription_services_privacy_notice.md [hunk 3]
```diff
@@ -1,55 +1,68 @@
...
 
-__Network data.__ When you activate Mozilla VPN, it will encrypt your internet traffic and send it to Mullvad. No logs of your network activity are maintained by either Mozilla nor our partner Mullvad when you're using the Mozilla VPN service.  Learn more at [Mullvad’s Privacy Policy](https://mullvad.net/help/no-logging-data-policy/).
+### Mozilla Monitor {: #monitor }
+
+__Interaction data.__ Mozilla receives data about your interactions with the Services. For instance, when you log in and out and the preferences you set; learn more about [Monitor interaction data](https://dictionary.telemetry.mozilla.org/apps/monitor_frontend). More generally, Mozilla may also use cookies, device information, and IP addresses, along with clear GIFs, cookies and third party services to help us understand in the aggregate how users engage with our products, services, communications, websites, online campaigns, snippets, devices, and other platforms. We use:
```
[the remark this answers is not in the document]
Comment (jejohnsonmoz) [reply en/subscription_services_privacy_notice.md:38]: Agree this should be lowercase s.

**Review by jejohnsonmoz — APPROVED** [review]

**Review by peiying2 — APPROVED** [review]

**Pull request merged**